tket1-passes/rust/Cargo.toml

```
@@ -0,0 +1,29 @@
+[package]
+name = "tket1-passes"
+version = "0.1.0"
```

Pre-release version (so the first one gets the 0.1.0 value)
```suggestion
version = "0.0.0"
```
Are we planning a separate versioning from the main `tket2` crates?

---

Yes, I don't think there will be very frequent updates to this crate (and it might be more in sync with tket1 than the rest of tket2...). I also don't think we should adopt the tket1 versioning, to give us the flexibility of releasing when useful to us.